roles/openshift_cloud_provider/tasks/openstack.yml

```diff
@@ -0,0 +1,13 @@
+- name: Configure Openstack integration
+  set_fact:
+    os_auth_url: "{{ lookup('env', 'OS_AUTH_URL') }}"
+    os_username: "{{ lookup('env', 'OS_USERNAME') }}"
+    os_password: "{{ lookup('env', 'OS_PASSWORD') }}"
+    os_tenant_id: "{{ lookup('env', 'OS_TENANT_ID') }}"
+    os_region: "{{ lookup('env', 'OS_REGION_NAME') }}"
```

Comment:
I'm wondering if we should allow setting these values based on variables passed in, instead of always using env lookups. I'd move these variables from set_fact and into roles/openshift_cloud_provider/defaults.yml:
```
ocp_os_auth_url: "{{ lookup('env', 'OS_AUTH_URL') }}"
ocp_os_username: "{{ lookup('env', 'OS_USERNAME') }}"
ocp_os_password: "{{ lookup('env', 'OS_PASSWORD') }}"
ocp_os_tenant_id: "{{ lookup('env', 'OS_TENANT_ID') }}"
ocp_os_region: "{{ lookup('env', 'OS_REGION_NAME') }}"
```

I've also added the 'ocp_' prefix since they would now be variables to be passed in.